catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -134,6 +134,10 @@ define([
                     if(this.get('valueUpperLon') && (this.get('valueUpperLon') > 180 || this.get('valueUpperLon')< -180)){
                         error = 'Invalid decimal longitude value. Must be -180 < lat < 180';
                     }
+                } else if(type === 'date'){
+                    if (isNaN(new Date(this.get('valueDate') + 'T' + this.get('valueTime') + 'Z'))) {
```

Comment:
So it seems that _.isDate returns true if the date is invalid. This means that _.isDate(new Date("anything")) always is true which made me think to rely on isNaN again. I found a thread where the underscore devs discussed and ultimately rejected adding an _.isValidDate method, but there is one present in an underscore-contrib library that performs two evaluations `_.isDate(x) && !_.isNaN(x.getTime()` I would still have to create the new Date object so if it doesn't have any issues, it might make some sense to leave this  with a single isNaN check.